src/stores/ConfigStore.ts

```diff
@@ -45,4 +57,13 @@ export const useConfigStore = createStore<ConfigState>('ConfigStore', (_, get) =
 
     return { cleengId, cleengSandbox, monthlyOfferId, yearlyOfferId };
   },
+  getInPlayerData: (): InPlayerData => {
```

Comment:
I think these methods (this and the cleeng one) can move to be internal methods in the respective services. They are really just syntactical-sugar helpers to reduce deep level property indexing on the config object...

Reply:
Removed the `InPlayer` part since it's not used yet. We can plan to refactor the cleeng part at later stage.